src/Dfc.CourseDirectory.Core/DataStore/Sql/QueryHandlers/GetProviderApprenticeshipQAStatusHandler.cs

```diff
@@ -19,6 +19,12 @@ public class GetProviderApprenticeshipQAStatusHandler :
 
             var paramz = new { query.ProviderId };
 
+#if DEBUG
+            if (System.Environment.GetEnvironmentVariable("BYPASS_QA")!=null && bool.Parse((System.Environment.GetEnvironmentVariable("BYPASS_QA"))))
```

Comment:
```suggestion
            if (bool.TryParse(System.Environment.GetEnvironmentVariable("BYPASS_QA", out var bypassQA) && bypassQA)
```

Reply:
Cheers. Rebased and switched to new out-var syntax.